src/icrar/leap-accelerate/model/cpu/MetaData.h

```diff
@@ -22,7 +22,12 @@
 
 #pragma once
 
-#include "cuda_runtime.h"
+#if __has_include(<cuda_runtime.h>)
```

Comment:
`#ifdef CUDA_ENABLED` aligns better with the rest of the code, and doesn't depend on a C++17 feature.